text/logs/0178-mapping-to-otlp-anyvalue.md

```diff
@@ -0,0 +1,167 @@
+# Mapping Arbitrary Data to OTLP AnyValue
+
+This document defines how to map (convert) arbitrary data (e.g. in-memory
+objects) to OTLP's AnyValue.
```

Comment:
Do we want to call out that this is specifically for the log datamodel's "body"?